core/iwasm/common/wasm_runtime_common.c

```diff
@@ -7827,43 +7842,27 @@ wasm_runtime_create_table(WASMModuleCommon *const module,
 
 #if WASM_ENABLE_INTERP != 0
     if (module->module_type == Wasm_Module_Bytecode) {
-        wasm_table_inst_t *out_table =
-            runtime_malloc(sizeof(wasm_table_inst_t), NULL, NULL, 0);
-        if (!out_table)
-            return NULL;
-
         WASMTableInstance *wasm_table =
             wasm_create_table((WASMModule *)module, type);
         if (!wasm_table) {
             wasm_runtime_free(wasm_table);
```

Comment:
had better remove this line, which frees a null pointer

Reply:
~ So confused that why I used a `wasm_runtime_free()` here? 😕